### PR TITLE
[PRM-2395] Allow users to reload the confirmation page

### PR DESCRIPTION
**Pull request by JBlacklaws**

### CODE REVIEW

**Comment by JBlacklaws** [comment]
https://user-images.githubusercontent.com/58850837/225652841-5bf3b2b0-a78c-404c-9921-5d949d0580bc.mov

File: app/controllers/CheckYourAnswersController.scala [hunk 1]
```diff
@@ -122,25 +122,38 @@ class CheckYourAnswersController @Inject()(checkYourAnswersPage: CheckYourAnswer
...
-          _ => Redirect(controllers.routes.CheckYourAnswersController.onPageLoadForConfirmation())
+          _ => Redirect(controllers.routes.CheckYourAnswersController.onPageLoadForConfirmation()).addingToSession(
+            List(
+              ("confirmationAppealType", confirmationAppealType),
+              ("confirmationStartDate", confirmationStartDate),
+              ("confirmationEndDate", confirmationEndDate),
+              ("confirmationObligation", confirmationObligation),
+              ("confirmationMultipleAppeals", confirmationMultipleAppeals),
+              ("confirmationIsAgent", confirmationIsAgent)): _*
+          )
```
Comment (AndrewRButcher) [inline app/controllers/CheckYourAnswersController.scala:140]:
```suggestion
            List(
              (SessionKeys.confirmationAppealType, confirmationAppealType),
              (SessionKeys.confirmationStartDate, confirmationStartDate),
              (SessionKeys.confirmationEndDate, confirmationEndDate),
              (SessionKeys.confirmationObligation, confirmationObligation),
              (SessionKeys.confirmationMultipleAppeals, confirmationMultipleAppeals),
              (SessionKeys.confirmationIsAgent, confirmationIsAgent)): _*
          )
```

**Review by AndrewRButcher — CHANGES_REQUESTED** [review]
One small change suggested

**Comment by AndrewRButcher** [comment]
Also needs rebasing

**Review by AndrewRButcher — APPROVED** [review]

**Pull request merged**